modules/Bio/EnsEMBL/Compara/RunnableDB/PairAligner/ChunkAndGroupDna.pm

```diff
@@ -135,7 +135,6 @@ sub create_chunks {
     } else {
         my $dnafrag_list = $dnafrag_dba->fetch_all_by_GenomeDB(
             $genome_db,
-            -COORD_SYSTEM_NAME  => 'toplevel',
```

Comment:
Is it safe to remove this? I know that newer cores are coming with the `primary_assembly` coord system, but what if we run the pipeline with an older core?

Reply:
This PR is tricky because it comes from comments in a closed PR. Here is the comment from @muffato that made me change this bit: https://github.com/Ensembl/ensembl-compara/pull/168#discussion_r384217116

Reply:
`-COORD_SYSTEM_NAME  => 'toplevel' is ignored by `fetch_all_by_GenomeDB`, cf https://github.com/Ensembl/ensembl-compara/blob/release/99/modules/Bio/EnsEMBL/Compara/DBSQL/DnaFragAdaptor.pm#L223